cluster/environments/azure-single-keyvault/main.tf

```diff
@@ -43,6 +43,14 @@ module "aks-gitops" {
   oms_agent_enabled        = "${var.oms_agent_enabled}"
 }
 
+module "acr" {
```

Comment:
Should we move this to it's own file since we will likely be expanding on some of this?

Otherwise LGTM

Reply:
Sounds good!